shared/getting-started/getDeviceOnDash/raspberrypi4.md

```diff
@@ -1,4 +1,4 @@
-Insert the SD card into your **{{ $device.name }}** and connect the ethernet cable if necessary. Now power up the **{{ $device.name }}** by inserting the micro USB cable.
+Insert the SD card into your **{{ $device.name }}** and connect the ethernet cable if necessary. Now power up the **{{ $device.name }}** by inserting the USB-C cable.
```

Comment:
I wonder if this might be another thing that needs to be "variable-ised"? 
`Now power up the **{{ $device.name }}** by inserting the {{ $device.cabletype }} cable.` :wink: